effects/gang_effect.py

```diff
@@ -0,0 +1,52 @@
+import message_text_config as msg
+import utils
+from citizens.citizen import Citizen
+from citizens.player import Player
+from game import Game
+
+from effects.effect import Effect, EffectStatus, InputStatusCode
+
+
+class GangEffect(Effect):
+    def __init__(self, context: 'Context', name: str,
+                 creator: Citizen) -> None:
+        super().__init__(context, name, creator, 13)
+
+    def _activate_impl(self) -> bool:
+        self.targets.append(self.creator)
+        self.user_interaction.save_active(msg.GangMessages.ACTIVATION_SUCCESS)
+
+        return True
+
+    def _resolve_impl(self) -> bool:
+        if self.game.round_number == self.activation_round:
+            return False
+
+        card_effects_list = []
```

Comment:
Don't use plural form twice in the name.

It's either card_effect_list
Or card_effects

Don't use cars_effectS_LIST